hyppo/multivariate/tests/test_dhsic.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_almost_equal
+
+from ...tools import linear, power
+from .. import Dhsic
+
+
+class TestDhsicStat:
+    @pytest.mark.parametrize("n, obs_stat", [(100, 0.04561), (200, 0.03911)])
+    @pytest.mark.parametrize("obs_pvalue", [1 / 1000])
+    def test_linear_oned(self, n, obs_stat, obs_pvalue):
+        np.random.seed(123456789)
+        x, y = linear(n, 1)
+        stat, pvalue = Dhsic(**{'gamma': 0.5}).test(*(x, y))
+
+        assert_almost_equal(stat, obs_stat, decimal=2)
+        assert_almost_equal(pvalue, obs_pvalue, decimal=2)
+
```

Comment:
you need unit tests for the error checking functions like `_multi_check_kernmat`. See the tests module in `hyppo.tools` for examples

Reply:
Added test_error_multidistkern to test_common.py using _multi_check_kernmat in common.py!